sonar-scanner-engine/src/main/java/org/sonar/batch/deprecated/DeprecatedSensorContext.java

```diff
@@ -43,19 +42,19 @@
 import org.sonar.api.resources.File;
 import org.sonar.api.resources.Project;
 import org.sonar.api.resources.Resource;
-import org.sonar.api.utils.SonarException;
+import org.sonar.batch.index.DefaultIndex;
 import org.sonar.batch.sensor.DefaultSensorContext;
 import org.sonar.batch.sensor.coverage.CoverageExclusions;
 
 public class DeprecatedSensorContext extends DefaultSensorContext implements SensorContext {
 
   private static final Logger LOG = LoggerFactory.getLogger(DeprecatedSensorContext.class);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOG" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1068)